src/plugins/charts/bar/BarGraphView.vue

```diff
@@ -144,11 +144,7 @@ export default {
             });
         },
         removeFromComposition(telemetryObject) {
-            let composition = this.domainObject.composition.filter(id =>
-                !this.openmct.objects.areIdsEqual(id, telemetryObject.identifier)
-            );
-
-            this.openmct.objects.mutate(this.domainObject, 'composition', composition);
+            this.openmct.composition.remove(telemetryObject);
```

Comment:
This function doesn't exist on the composition API. I think you mean `this.composition.remove(telemetryObject)`

Reply:
What! How did it not fail when I tested this!? Jesus! :)